File: models/AutenticacaoModel.dart
class AutenticacaoModel {
  String? id;
  String? senha;
  String? tokenAcesso;

  AutenticacaoModel({
    this.id,
    this.senha,
    this.tokenAcesso,
  });

  factory AutenticacaoModel.fromJson(Map<String, dynamic> json) {
    return AutenticacaoModel(
        id: json['id'].toString(), tokenAcesso: json['access_token']);
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['id'] = this.id;
    data['senha'] = this.senha;
    return data;
  }
}

File: angular-gpp/src/app/models/AutenticacaoModel.ts
export class AutenticacaoModel {
  id: string;
  senha: string;
  tokenAcesso: string;

  constructor(id: string, senha: string, tokenAcesso: string) {
    this.id = id;
    this.senha = senha;
    this.tokenAcesso = tokenAcesso;
  }
}
